### DISCUSSION TITLE
Canal RCN

### DISCUSSION THREAD

**Discussion by nomadmzl** [iptv-org/community#848] [Q&A]
### Channel ID

CanalRCN.co

### Notes

Please, add it to [Colombia m3u8](https://iptv-org.github.io/iptv/countries/co.m3u) list. Thanks

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
https://www.canalrcn.com/senal-en-vivo/
dailymotion stream and geo

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19795
